src/status_im/ui/components/connectivity/view.cljs

```diff
@@ -6,6 +6,22 @@
             [status-im.ui.components.connectivity.styles :as styles]
             [status-im.i18n :as i18n]))
 
+(defview error-label
+  [{:keys [view-id label mailserver-error?] :as opts}]
+
+  {:should-component-update
```

Comment:
@rasom  could you please elaborate ?  this fix looks complex , thanks

Reply:
https://github.com/status-im/status-react/pull/6068/files#diff-df3f46463da5ee34e009067a466d7039R17 at this line depending on `view-id` we pass `modal?` parameter to `styles/text-wrapper`. This `modal?` parameter affects `top` property of the style. 

When we navigate back from `:chat-modal` to any other screen `:view-id` in `app-db` gets changed while the animation is in progress, which causes the change of `top` position before the screen is hidden from the user. In result "Fetching messages..." is flickering. 

You can easily reproduce this on android device/emulator by removing `:should-component-update` handler.